packages/contracts-bedrock/scripts/deploy/Deploy.s.sol

```diff
@@ -238,13 +238,17 @@ contract Deploy is Deployer {
 
         console.log("Deploying a fresh OP Stack with existing SuperchainConfig and ProtocolVersions");
 
+        // Set msg.sender to zero address because the proxies will return implementation only for
+        // zero address or admin.
+        vm.startPrank(0x0000000000000000000000000000000000000000);
```

Comment:
```suggestion
        vm.startPrank(address(0));
```

Reply:
Another option would be to use the erc1967 library to read the slot directly from storage. This works as well, i am fine to merge with the nit fixed